techniques/css/C37.html

```diff
@@ -2,86 +2,66 @@
-               layout can be wider or smaller at different points. This technique ensures images
-               do not break out of their layout area, including in one-column layouts where it would
-               cause scrolling.
-            </p>
-            <p>The basic principles of fitting images are to:</p>
+            <h2>解説</h2>
+            <p>この達成方法の目的は、320 CSS ピクセルに相当する幅において横スクロールバーを、又は 256 CSS ピクセルに相当する高さにおいて縦スクロールバーを導入することなく、画像を表示できるようにすることである。これは、画像の元のサイズを維持しつつ使用可能なスペースに適応させる、CSS の <code>max-width</code> 及び <code>height</code> プロパティ手法を用いることによって行われる。</p>
+            <p>レスポンシブなレイアウトでは、カラムやレイアウトブロックを追加したり削除したりできる。また、レイアウトの各部分は、さまざまなポイントで幅を広くしたり小さくしたりできる。この手法により、スクロールが発生する1カラムのレイアウトを含めて、画像がレイアウト領域からはみ出すことがなくなる。</p>
+            <p>画像を適応させるための基本原則は次のとおりである:</p>
             <ol>
                
-               <li>Define the <code>max-width</code> property for images, and;
+               <li>画像の <code>max-width</code> プロパティを定義する。
                </li>
                
-               <li>Define the <code>height</code> property for images, so they enlarge or shrink in the available space and respond
-                  to zoom levels.
+               <li> 画像の <code>height</code> プロパティを定義して、使用可能なスペースで画像を拡大又は縮小し、ズームレベルに対応させる。  
                </li>
                
             </ol>
-            <p>All images require design finesse by making sure the original size fits the biggest
-               size of the available spaces to achieve good-looking results at a wide range of viewport
-               sizes and zoom levels.
+            <p>全ての画像は、さまざまなビューポートサイズとズームレベルで綺麗な見ためになるようにするために、元のサイズが利用可能なスペースの最大サイズに確実に適応させるようにすることによる、デザインの精巧さを必要とする。
             </p>
          </section>
          <section id="examples">
-            <h2>Examples</h2>
+            <h2>事例</h2>
             <section class="example" id="example-1-fitting-images-in-html-and-css">
-               <h3>Example 1: Fitting images in HTML and CSS</h3>
-               <p>The following simple example uses HTML and CSS to create a fitting image. The layout
-                  regions adjust their size as the viewport is adjusted. The images subsequently adjust
-                  their size to fit within the layout region containers.
+               <h3>例 1: HTML と CSS での画像の適応</h3>
+               <p>次の簡単な例では、適応した画像を設置するために HTML と CSS を使用する。ビューポートが調整されると、レイアウト領域は同じようにサイズが調整される。その後、画像はレイアウト領域のコンテナ内に収まるようにサイズが調整される。
                </p>
-               <p>The zoom level can be increased to 400% without requiring scrolling in more than one
-                  direction. This particular example uses a percent size for the <code>max-width</code> and auto size for the <code>height</code> of the image to remain the original dimensions.
+               <p>複数の方向にスクロールしなくても、ズームレベルを400％まで上げることができる。この例では、画像の <code>max-width</code> にパーセントサイズを使用し、<code>height</code> には自動（auto）サイズを使用することにより、元のサイズを維持している。
```

Comment:
コード中に auto があるわけではないので削除しておきます。

```suggestion
               <p>複数の方向にスクロールしなくても、ズームレベルを400％まで上げることができる。この例では、画像の <code>max-width</code> にパーセントサイズを使用し、<code>height</code> には自動サイズを使用することにより、元のサイズを維持している。
```